src/main/resources/static/main.css

```diff
@@ -5,4 +5,12 @@
     margin-bottom: 2rem;
     display: block;
     box-sizing: border-box;
-}
\ No newline at end of file
+}
+
+.outerContentWrapper{
+    display: block;
+    position: relative;
+    max-height: 80vh;
+    height: 53vh;
+    overflow-y: scroll;
+}
```

Comment:
Added CSS class to constrain the content of the page to keep it from interfering with other elements such as the footer.